domain/schema_validator.go

```diff
@@ -209,7 +220,35 @@ func (s *schemaValidator) Check(txnTS uint64, schemaVer int64, relatedTableIDs [
 
 func (s *schemaValidator) enqueue(schemaVersion int64, relatedTableIDs []int64) {
 	s.deltaSchemaInfos = append(s.deltaSchemaInfos, deltaSchemaInfo{schemaVersion, relatedTableIDs})
-	if len(s.deltaSchemaInfos) > maxNumberOfDiffsToLoad {
+	s.notMergeCnt++
+
+	maxCnt := int(variable.GetMaxDetalSchemaCount())
+	if len(s.deltaSchemaInfos) > maxCnt && s.notMergeCnt > maxCnt/2 {
+		s.merge()
+		s.notMergeCnt = 1
+	}
+	if len(s.deltaSchemaInfos) > maxCnt {
 		s.deltaSchemaInfos = s.deltaSchemaInfos[1:]
 	}
 }
+
+func equal(a, b []int64) bool {
```

Comment:
https://godoc.org/bytes#Equal

Reply:
@tiancaiamao There are different types. The argument types of `equal` are `[]int64`.